clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestLabeler.java

```diff
@@ -20,6 +20,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.netflix.spinnaker.moniker.Moniker;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Somehow this comment got lost from my review:

`org.apache.commons.lang3.StringUtils` is the one we have in `spinnaker-dependencies`; I think this is just transitively included.

Reply:
Fixed.